cibuildwheel/__main__.py

```diff
@@ -136,11 +136,19 @@ def main():
     )
 
     if platform == 'linux':
-        manylinux1_x86_64_image = os.environ.get('CIBW_MANYLINUX1_X86_64_IMAGE', None)
-        manylinux1_i686_image = os.environ.get('CIBW_MANYLINUX1_I686_IMAGE', None)
+        # TODO Check CIBW_MANYLINUX1_X86_64_IMAGE and CIBW_MANYLINUX1_I686_IMAGE environment variables and error if they exist
+        # TODO Check CIBW_BUILD and CIBW_SKIP for "manylinux1" and error if present
+        # TODO Add documentation on CIBW_ENVIRONMENT and possibility of AUDITWHEEL_PLAT if not defined by custom manylinux image
```

Comment:
I agree - there need to be breaking changes in this version, so let's detect obsolete options and error here, directing people to the relevant part of the changelog.